extension_templates/annotation.py

```diff
@@ -179,3 +183,30 @@ def _update(self, X, Y=None):
 
         # implement here
         # IMPORTANT: avoid side effects to X, fh
+
+    # todo: return default parameters, so that a test instance can be created
+    @classmethod
+    def get_test_params(cls):
+        """Return testing parameter settings for the estimator.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, (i.e),
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`
+        """
+
+        # todo: set the testing parameters for the estimators
+        # Testing parameters can be dictionary or list of dictionaries
+        #
+        # example 1: specify params as dictionary
+        # any number of params can be specified
+        # params={"est":value0, "parama":value1, "paramb":value2}
```

Comment:
please adhere to linting rules in the example code , should be `params = {"est": value0, ` and so on

Reply:
@fkiraly I have made following changes
Added space around = and a space after : in dictionary wherever the code has been written in comments
Is there any other changes apart from these?

Reply:
hm, I can't see these changes in here - have you pushed them?

Reply:
I have pushed them now. @fkiraly could you please have a look?